src/main/resources/templates/team/main.html

```diff
@@ -45,16 +45,15 @@ <h2>Tým:</h2>
             </td>
         </tr>
         <tr>
-            <td>Počet členů:(*)</td>
+            <td>Počet členů: (*)</td>
             <td><input type="number" th:field="*{numOfMembers}"/></td>
             <td th:if="${#fields.hasErrors('numOfMembers')}" th:errors="*{numOfMembers}"
                 style="color: orangered; font-size: 14px">Chyba - Počet členů
             </td>
         </tr>
         <tr>
             <td></td>
-            <td><input type="submit" th:value="${action=='/team/add'?'Nový':'Uložit'}"/>
-                <input type="reset" value="Vynulovat"/>
+            <td><input type="submit" th:value="${action=='/team/add'? 'Nový':'Uložit'}"/>
```

Comment:
Tady bych když tak přidal už mezery všude kolem těch operátorů:
'/team/add' ? 'Nový' : 'Uložit'

Reply:
mezery pridany